src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/wwwroot/index.html

```diff
@@ -0,0 +1,92 @@
+              integrity="sha256-eSi1q2PG6J7g7ib17yAaWMcrr5GrtohYChqibrV7PBE="/>
+    <link rel="stylesheet" href="/css/site.css" />
+</head>
+<body>
+
+    <header>
+        <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-light bg-white border-bottom box-shadow mb-3">
+            <div class="container">
+                <a class="nav-link text-dark" href="/index.html">Chat with SignalR</a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".navbar-collapse" aria-controls="navbarSupportedContent"
+                        aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <ul class="nav navbar-nav navbar-right">
+                    <li><span>Server Time: </span><span id="serverTime" style="width: 200px; display: inline-block;"></span></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <div class="container">
+        <main role="main" class="pb-3">
+            <div class="container">
+                <div class="row">&nbsp;</div>
+                <div class="row">
+                    <div class="col-12">
+                        <form class="form-inline">
+                            <div class="input-prepend input-append">
+                                <input type="text" name="group-name" id="group-name" placeholder="Type a group name" />
+                                <input type="button" id="join-group" class="btn btn-sm btn-primary" value="Join Group" />
+                                <input type="button" id="leave-group" class="btn btn-sm btn-primary" value="Leave Group" />
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-12">
+                        <form class="form-inline">
+                            <div class="input-prepend input-append">
+                                <input type="text" name="group-message" id="group-message-text" placeholder="Type a message" />
+                                <input type="button" id="groupmsg" class="btn btn-sm btn-primary" value="Send to Group" />
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-12">
+                        <hr />
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-6">&nbsp;</div>
+                    <div class="col-6">
+                        <ul id="messagesList"></ul>
+                    </div>
+                </div>
+            </div>
+        </main>
+    </div>
+
+    <footer class="border-top footer text-muted">
+        <div class="container">
+            &copy; 2019 - SignalRTemplate
```

Comment:
This doesn't look like the right footer. See this for example:

https://github.com/aspnet/AspNetCore/blob/5e62914cde1c0cbdc9837add6190241c722d4abc/src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml#L56

Is this wrong in other templates?

Reply:
That footer is specific for an ASP.NET Razor page. this is an HTML page. the asp-area attribute would need to be omitted, etc. cool? adding full razor here seems like overkill.

Reply:
Yeah sorry I just mean to put the year/company name like we do in other templates. Otherwise in this template it'll always say 2019, and it'll say SignalRTemplate, instead of the project name.